### PR TITLE
Grammatical errors rectified

### PR DESCRIPTION
**Pull request by mariarose-francis**
The 546-bug-dnsrecon-tooltip task was completed. Rectified all the grammatical errors in DNSRecon tooltip. Please review the changes committed.
Thank you.

### CODE REVIEW

**Review by bradleybogg — CHANGES_REQUESTED** [review]
@mariarose-francis 

It is not acceptable to have no description.
Please amend this to reflect the changes you have completed. I will then proceed with review.

**Review by bradleybogg — DISMISSED** [review]
@mariarose-francis 

Please make sure you push your changes. I cannot see any changes as of yet. 

Thank you!

File: src/components/Dnsrecon/Dnsrecon.tsx [hunk 1]
```diff
@@ -7,16 +7,15 @@ import { UserGuide } from "../UserGuide/UserGuide";
...
-    "Using Dnsrecon:\n" +
-    "Step 1: Enter a Target Domain URL.\n" +
-    "       Eg: https://www.deakin.edu.au\n\n" +
-    "Step 2: Click Scan to commence Dnsrecon's operation.\n\n" +
-    "Step 3: View the Output block below to view the results of the tools execution.";
+    "DNSRecon is a Python script that has an extensive list of functions. This tool is primarily used for " +
+    "DNS enumeration and scanning, for example, it may enumerate DNS records, SRV records, and hosts and " +
+    "domains using Google.\n\nFurther uses for the tool can be found at: https://www.kali.org/tools/dnsrecon/\n\n" +
+    "Using DNSRecon:\n" +
+    "Step 1: Enter a Target Domain URL, for example, https://www.deakin.edu.au\n\n" +
```
Comment (bradleybogg) [inline src/components/Dnsrecon/Dnsrecon.tsx:16]: - [ ] please correct the capitalization of the words that is incorrect.

File: src/components/Dnsrecon/Dnsrecon.tsx [hunk 2]
```diff
@@ -7,16 +7,15 @@ import { UserGuide } from "../UserGuide/UserGuide";
...
+    "Step 2: Click Scan to commence DNSRecon's operation.\n\n" +
```
Comment (bradleybogg) [inline src/components/Dnsrecon/Dnsrecon.tsx:17]: - [ ] please correct the capitalization of the words that is incorrect.

File: src/components/Dnsrecon/Dnsrecon.tsx [hunk 3]
```diff
@@ -7,16 +7,15 @@ import { UserGuide } from "../UserGuide/UserGuide";
...
+    "Step 3: View the Output block below to view the results of the tool's execution.";
```
Comment (bradleybogg) [inline src/components/Dnsrecon/Dnsrecon.tsx:18]: - [ ] please correct the capitalization of the words that is incorrect.

**Review by bradleybogg — CHANGES_REQUESTED** [review]
Hi @mariarose-francis,

Thanks for your work.

I have requested some small changes so that you can see this process. The changes here still require some work to be finalized. The purpose of tooltips is to allow you to learn the process of committing and the company.

For this reason, after the changes I intend to approve and have the technical components completed under baselining. 

Thank you!

I recommend that you investigate the baseline task for this component on completion.

**Comment by mariarose-francis** [comment]
Closing this PR as I have created a new PR with changes committed.

**Pull request closed without merging**